dialogue-clients/metrics.md

```diff
@@ -68,6 +68,8 @@ Dialogue DNS metrics.
     - `success`: DNS resolution succeeded using `InetAddress.getAllByName`.
     - `fallback`: DNS resolution using the primary mechanism failed, however addresses were available in the fallback cache.
     - `failure`: No addresses could be resolved for the given hostname.
+- `client.dns.lookupError` (meter): DNS resolver query failures.
```

Comment:
let's match the `failure` wording above, perhaps naming this `client.dns.failure`

Reply:
fixed